toolkit/tools/internal/file/file.go

```diff
@@ -129,7 +129,7 @@ func Write(data string, dst string) (err error) {
 
 // Append appends a string to the end of file dst.
 func Append(data string, dst string) (err error) {
-	logger.Log.Debugf("Appending to (%s)", dst)
+	logger.Log.Debugf("Appending (%s) to (%s)", data, dst)
```

Comment:
I'd break it out into a separate line now and maybe add an indent to make it clear what is the appended data. In case we append nothing right now we have a slightly cryptic log, while if we append something long, then it's going to be hard to read. Not to mention that the appended data will blend with "Appending" and the "to (%s)" parts.

Example of my suggestion:
```
logger.Log.Debugf("Appending following string to (%s):\n\n\t%s", dst)
```

Reply:
Here is an example of the current output as taken from the debug log file:
```
time="2021-03-09T07:45:20Z" level=debug msg="Appending (UUID=779A-C79D /boot/efi vfat umask=0077 0 2\n) to (/installroot/etc/fstab)"
```
If we have an error and append nothing, we'd have the string `"Appending () to (<file>)"` which seems pretty clear. And it looks like multiline appends will just result in a longer single-line `data` string so I'm not sure if the suggestion will improve things. Maybe there's a different format to use to preserve a multiline string append?